isos/bootstrap.sh

```diff
@@ -81,7 +81,7 @@ else
 fi
 
 # copy in our components
-cp ${BIN}/tether-linux $(rootfs_dir $PKGDIR)/bin/tether
```

Comment:
please continue to call this tether-linux. We still have an intent to add other OS support.